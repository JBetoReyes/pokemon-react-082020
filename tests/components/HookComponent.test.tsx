import { shallow } from 'enzyme';
import React from 'react';
import HookApp from '@components/hooks/HookApp.component';

describe('HookComponent', () => {
  const hookAppJSX = <HookApp />;
  let wrapper = shallow(hookAppJSX);

  beforeEach(() => {
    wrapper = shallow(hookAppJSX);
  });

  test('The component snapshot should match', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
