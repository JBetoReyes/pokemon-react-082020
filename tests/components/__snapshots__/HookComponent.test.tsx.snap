// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookComponent The component snapshot should match 1`] = `
<Fragment>
  <div
    className=""
  >
    <h1>
      hello
    </h1>
  </div>
</Fragment>
`;
